src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Михаил
+  Date: 08.09.2020
+  Time: 16:27
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
see checklist. We should remove this comment

Reply:
done

Reply:
comments are still here 🙂

Reply:
done